include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -876,6 +876,10 @@ def SpecializeOption : Pass<"firrtl-specialize-option", "firrtl::CircuitOp"> {
   }];
   let constructor = "circt::firrtl::createSpecializeOptionPass()";
 
+  let options = [
+    Option<"selectDefault", "specialize-to-default-if-no-override", "bool", "false",
```

Comment:
Can the option name (both CLI and variable) be more descriptive than just "specialize"? This is a term that is used also for layers and could mean (roughly) and removal of a parameter. Or: include instance choice in the name.

Reply:
Renamed both to be a bit more descriptive. Hopefully its not too long,
`select-default-for-unspecified-instance-choice`

Reply:
SGTM. Overly descriptive and obvious is fine-by-me. 💯